AndroidXCI/lib/build.gradle.kts

```diff
@@ -55,6 +54,8 @@ dependencies {
  * Copy the empty apk into our build which will be used for library integration tests.
  * Unfortunately, FTL always requires a test apks, hence we need this fake one.
  */
+// specify dependency so that we can access the packageDebug task from the placeholder.
+evaluationDependsOn(":placeholderapp")
```

Comment:
This whole thing is not the best way to do this, we should be exporting this APK as an artifact and on this side consuming it as input, but i'm not blocking on that.

Reply:
thanks for unblocking it. Idk how to do the artifact part but if you have some pointers, i can work on a followup.